baseplate/observers/sentry.py

```diff
@@ -155,13 +160,23 @@ class SentryUnhandledErrorReporter:
     def __init__(self, hub: "Hub", client: raven.Client):
         self.original_print_exception = getattr(hub, "print_exception")
         self.raven = client
+        self.logger = logging.getLogger(__name__)
```

Comment:
🔕 OOC, why set `self.logger` here rather than doing a global `logger` at the top of the file?

Reply:
The hub stuff has to be super defensive about interpreter shutdown where globals can suddenly be `None`d out on you. I'm being a little cargo-culty here but didn't want to take the risk of a crash in weird situations and since this is only instantiated once per app there shouldn't be any cost to doing it this way.